contracts/BatchAuction.sol

```diff
@@ -19,4 +29,16 @@ contract BatchAuction {
         publicKeyToAccountMap[msg.sender] = accountId;
         accountToPublicKeyMap[accountId] = msg.sender;
     }
+
+    function addToken(address _tokenAddress) public onlyOwner() {
+        require(tokenAddresToIdMap[_tokenAddress] == 0, "Token already registered!");
+        require(numTokens + 1 <= MAX_TOKENS, "Token id exceeds max tokens");
+
+        tokenAddresToIdMap[_tokenAddress] = numTokens + 1;
+        tokenIdToAddressMap[numTokens + 1] = _tokenAddress;
+
+        numTokens++;
```

Comment:
Interesting. We could have a similar model for accounts, so that you don’t have to specify the slot ID specifically.

Reply:
Yes, I wouldn't mind implementing that. Although I would rather make that change in a different PR.